src/mne_lsl/lsl/load_liblsl.py

```diff
@@ -0,0 +1,232 @@
+        c_void_p,
+    ]
+    lib.lsl_parent.restype = c_void_p
+    lib.lsl_parent.argtypes = [
+        c_void_p,
+    ]
+    lib.lsl_child.restype = c_void_p
+    lib.lsl_child.argtypes = [c_void_p, c_char_p]
+    lib.lsl_next_sibling_n.restype = c_void_p
+    lib.lsl_next_sibling_n.argtypes = [c_void_p, c_char_p]
+    lib.lsl_previous_sibling_n.restype = c_void_p
+    lib.lsl_previous_sibling_n.argtypes = [c_void_p, c_char_p]
+    lib.lsl_name.restype = c_char_p
+    lib.lsl_name.argtypes = [
+        c_void_p,
+    ]
+    lib.lsl_value.restype = c_char_p
+    lib.lsl_value.argtypes = [
+        c_void_p,
+    ]
+    lib.lsl_child_value.restype = c_char_p
+    lib.lsl_child_value.argtypes = [
+        c_void_p,
+    ]
+    lib.lsl_child_value_n.restype = c_char_p
+    lib.lsl_child_value_n.argtypes = [c_void_p, c_char_p]
+    lib.lsl_append_child_value.restype = c_void_p
+    lib.lsl_append_child_value.argtypes = [c_void_p, c_char_p, c_char_p]
+    lib.lsl_prepend_child_value.restype = c_void_p
+    lib.lsl_prepend_child_value.argtypes = [c_void_p, c_char_p, c_char_p]
+
+    # return type for lsl_set_child_value, lsl_set_name, lsl_set_value is int
+    lib.lsl_set_child_value.argtypes = [c_void_p, c_char_p, c_char_p]
+    lib.lsl_set_name.argtypes = [c_void_p, c_char_p]
+    lib.lsl_set_value.argtypes = [c_void_p, c_char_p]
+    lib.lsl_append_child.restype = c_void_p
+    lib.lsl_append_child.argtypes = [c_void_p, c_char_p]
+    lib.lsl_prepend_child.restype = c_void_p
+    lib.lsl_prepend_child.argtypes = [c_void_p, c_char_p]
+    lib.lsl_append_copy.restype = c_void_p
+    lib.lsl_append_copy.argtypes = [c_void_p, c_void_p]
+    lib.lsl_prepend_copy.restype = c_void_p
+    lib.lsl_prepend_copy.argtypes = [c_void_p, c_void_p]
+    lib.lsl_remove_child_n.argtypes = [c_void_p, c_char_p]
+    lib.lsl_remove_child.argtypes = [c_void_p, c_void_p]
+    lib.lsl_destroy_string.argtypes = [c_void_p]
+
+    # TODO: Check if the minimum version for MNE-LSL requires those try/except.
+    try:
+        lib.lsl_pull_chunk_f.restype = c_long
+        lib.lsl_pull_chunk_d.restype = c_long
+        lib.lsl_pull_chunk_l.restype = c_long
+        lib.lsl_pull_chunk_i.restype = c_long
+        lib.lsl_pull_chunk_s.restype = c_long
+        lib.lsl_pull_chunk_c.restype = c_long
+        lib.lsl_pull_chunk_str.restype = c_long
+        lib.lsl_pull_chunk_buf.restype = c_long
+    except Exception:  # pragma: no cover
+        logger.info(
+            "[LIBLSL] Chunk transfer functions not available in your LIBLSL version."
+        )
+    try:
+        lib.lsl_create_continuous_resolver.restype = c_void_p
+        lib.lsl_create_continuous_resolver_bypred.restype = c_void_p
+        lib.lsl_create_continuous_resolver_byprop.restype = c_void_p
+    except Exception:  # pragma: no cover
+        logger.info(
+            "[LIBLSL] Continuous resolver functions not available in your LIBLSL "
+            "version."
+        )
```

Comment:
Now that we bundle and conda-forge is probably sufficiently up to date, maybe we don't need these?

Reply:
Someone could provide an old `liblsl` in the environment variable; so even if it's unlikely, it doesn't hurt to have this try/except caught this edge case.